scripts/ralph/acceptance-prd224.sh

```
@@ -0,0 +1,94 @@
+
+# Backend gate is BRANCH-SCOPED (2026-08-27 amendment): ~49 pre-existing
+# environmental fails/errors in the full local suite — CI test.yml is the
+# full-suite gate. Locally we prove THIS PRD's own tests green.
+echo ""
+echo "── branch-scoped backend tests (this PRD's tests; full-suite provenance = CI test.yml)"
+BASEP=$(git merge-base HEAD "$BASE_BR" 2>/dev/null || git merge-base HEAD origin/main 2>/dev/null || echo "")
+CHT=$(git diff --name-only "${BASEP:-HEAD~40}"..HEAD -- orchestrator 2>/dev/null | grep -E '(^|/)tests/.*\.py$' | sed 's|^orchestrator/||' | tr '\n' ' ')
+if [ -z "${CHT// /}" ]; then
+  echo "   ✅ PASS: no backend test files changed on this branch (CI covers the full suite)"
+else
+  if ( cd orchestrator && python3 -m pytest --timeout=90 --timeout-method=thread -o faulthandler_timeout=120 -p no:cacheprovider -q $CHT ); then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)